src/main/java/com/database/data/JdbcConnectionUtil.java

```diff
@@ -0,0 +1,29 @@
+package com.database.data;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+import java.util.logging.Logger;
+
+public class JdbcConnectionUtil {
+    private static final Logger LOGGER = Logger.getLogger(JdbcConnectionUtil.class.getName());
+    private static final String DB_DRIVER = "com.mysql.cj.jdbc.Driver";
+
+    public static Connection getConnection() {
+        Connection dbConnection = null;
+        try {
+            FileInputStream fis;
+            Properties property = new Properties();
+            fis = new FileInputStream("src/main/resources/config.properties");
+            property.load(fis);
+            Class.forName(DB_DRIVER);
```

Comment:
This line is a redundant  one
![image](https://user-images.githubusercontent.com/20267148/55959752-10d70700-5c74-11e9-9871-8cb0716ae4db.png)